docs/source/using/experimental-features.adoc

```diff
@@ -152,6 +152,170 @@ To get the status of the DNS server:
 $ minishift dns status
 ----
 
+[[local-dns-setup-macos]]
+== Local DNS Setup for macOS
```

Comment:
![localdnsmacos](https://user-images.githubusercontent.com/3125299/41415356-fde4f69e-7005-11e8-991e-64e9accb9f2e.png)
This section of docs should be one level below `Local DNS Server`. It should start with `===`.